lib/windows.js

```diff
@@ -108,7 +137,8 @@ function windows() {
 			y: bounds.top,
 			width: bounds.right - bounds.left,
 			height: bounds.bottom - bounds.top
-		}
+		},
+		memoryUsage: memoryCounters.WorkingSetSize === 0 ? undefined : memoryCounters.WorkingSetSize
```

Comment:
I don't really see why Windows every would disallow getting memory usage. Is it documented somewhere that this can not work in some instances?

Reply:
Windows system is extremely convoluted, especially when we go for old api's going all the way to winxp, so the posibilities are basically endless(which id kinda why it's so full of security vulnerabilities). However, I imagine we might have troubles reading it if it's "run as administrator" process and we're not. Also, there is a restriction sandbox(applied to new and downloaded from internet applications and maybe something else) which is barely documented, and will probably prevent the process from reading such info about any process in the system. Plus, depending on what local administrator configured there might be other cases.

Reply:
To be frank, even the solution I implemented for reading memory might not work on some versions/builds of windows on the older side. (To fully grasp it, as far as I understand, one needs to read quite a lot of sources and probably access some undocumented apis.) Though, I expect it work at most if not all versions.